_includes/manuals/2.1/2.1_quickstart_installation.md

```diff
@@ -1,6 +1,6 @@
 [The Foreman installer](/manuals/{{page.version}}/index.html#3.2ForemanInstaller) uses Puppet **(4 or later required)** to install Foreman. This guide assumes that you have a newly installed operating system, on which the installer will setup Foreman, a Puppet master, and the [Smart Proxy](/manuals/{{page.version}}/index.html#4.3SmartProxies) by default. It's **not advisable** to follow the steps below on an existing system, since the installer will affect the configuration of several components.
 
-**If you're looking for a special scenario such as Katello content management, please follow the [Katello]({{ site.baseurl }}plugins/katello) installation instructions**
+**If you want to manage content with Foreman or you're a new user unfamiliar with all of Foreman's capabilities, please follow the [Katello]({{ site.baseurl }}plugins/katello) installation instructions. If you do not install Katello with Foreman, you cannot add it at a later stage**
```

Comment:
I think content is too vague. If you don't know this means RPMs or other things, it means nothing. Other forms of content management can also be just doing a yum update on hosts (which REX without Katello can do).

I'd also say "Installing Katello on top of an existing Foreman is unsupported". It has been done by users. https://community.theforeman.org/t/rfc-adding-katello-to-a-foreman-install/18060 is a proposal to make this supported by adding checks for the caveats, but advanced users know how to work around that already. Of course, you wouldn't list all of that, just say "unsupported" instead of "cannot".

Reply:
Thanks!